haystack/nodes/reader/table.py

```diff
@@ -440,10 +417,7 @@ def postprocess(self, model_outputs):
                     else:
                         answer_str = self._aggregate_answers(aggregator, cells)
                     current_score = answer_scores[index]
-                    if self.return_table_cell:
-                        answer_offsets = _calculate_answer_offsets(ans_coordinates_per_table)
-                    else:
-                        answer_offsets = _calculate_answer_offsets_span(ans_coordinates_per_table, string_table)
+                    answer_offsets = _calculate_answer_offsets_span(ans_coordinates_per_table, string_table)
```

Comment:
```suggestion
                    answer_offsets = _calculate_answer_offsets(ans_coordinates_per_table)
```
Now `return_table_cell` default value is `True`.
Am I correct?